src/test/java/no/unit/bibs/contents/document/handlers/CreateDocumentTest.java

```diff
@@ -0,0 +1,114 @@
+package no.unit.bibs.contents.document.handlers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import no.unit.bibs.contents.document.DocumentDao;
+import no.unit.bibs.contents.document.DocumentDto;
+import no.unit.bibs.contents.document.DocumentTestBase;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.net.HttpURLConnection;
+import java.nio.file.Path;
+import java.time.Instant;
+
+import static no.unit.bibs.contents.CreateContentsApiHandlerTest.CREATE_CONTENTS_EVENT;
+import static no.unit.nva.commons.json.JsonUtils.dtoObjectMapper;
+import static nva.commons.core.ioutils.IoUtils.stringFromResources;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+class CreateDocumentTest extends DocumentTestBase {
+
+    private CreateHandler testHandler;
+
+    @BeforeEach
+    void setUp() throws JsonProcessingException, BadRequestException {
+        init();
+        testHandler = new CreateHandler(documentService, environment);
+        var dto = dtoObjectMapper.readValue(RESOURCE_PATH, DocumentDto.class);
+        when(mockedClient.create(any()))
+            .thenReturn(dto.toDaoBuilder().created(Instant.now()).modified(Instant.now()).build());
+    }
+
+
+    @Test
```

Comment:
you should call processInput first, asserting no exception is thrown. also, rename the test to ...returnsCreated, not ..returnsOk

Reply:
No, this function always return the same (200), it is basically pointless to test, but if we want 100% coverage, this is what we have to do.